olive/extra_dependencies.json

```diff
@@ -33,13 +33,10 @@
         "torch-tensorrt"
     ],
     "lora": [
-        "accelerate",
-        "peft"
-    ],
-    "qlora": [
         "accelerate",
         "bitsandbytes",
-        "peft"
+        "peft",
```

Comment:
I have some upcoming changes to the lora passes which will make these requirements cleaner. If you are okay with it, I can do the change here and in `run.py` in my PR so that this PR only deals with the data configs part 
https://github.com/microsoft/Olive/compare/main...jambayk/ort-bf16

Reply:
yes, better to split the unrelated changes to another PR.

Reply:
Sure, please go ahead for your changes and PR.